yalelibrary/summon-citation_export
# Base level functionality now available

The three types of export type are now available and this seems like a good basis for creating the 'test' branch base and then, perhaps, creating new feature branches as needed.

Testing steps:

1. Add this Gemfile branch to search-frontend:
    `gem 'summon_citation-export', :git => 'git@github.com:yalelibrary/summon-citation_export.git', branch: 'start'`
2. Add the requirements in `search-frontend/app/controllers/application_controller.rb`:
    `require summon/refworks`
    `require summon/ris`
    `require summon/endnote`
3. Put a debug flag in the `search_frontend/app/models/spectrum/search_engines/summon.rb` file on line 132.
4. Do an Articles+ search with server running
5. Try using the functions on `@search.documents[0].to_refworks` and `@search.documents[0].to_refworks_text`
6. Repeat for other document types (ris, endnote)
7. Verify data looks roughly correct
8. Remove debugger flag

Once we feel that looks good we can merge this code into `test` and call it the base code to be modified from other feature branches/PR's.
